cfgov/unprocessed/js/routes/owning-a-home/common.js

```diff
@@ -2,20 +2,18 @@
    Common application-wide scripts for owning-a-home.
    ========================================================================== */
 
-var FormSubmit = require( '../../organisms/FormSubmit.js' );
-var validators = require( '../../modules/util/validators' );
-var ratingsForm = require( '../../apps/owning-a-home/ratings-form' );
-var BASE_CLASS = 'o-email-signup';
-var emailSignup = document.body.querySelector( '.' + BASE_CLASS );
+const FormSubmit = require( '../../organisms/FormSubmit.js' );
+const validators = require( '../../modules/util/validators' );
+const ratingsForm = require( '../../apps/owning-a-home/ratings-form' );
+const BASE_CLASS = 'o-email-signup';
+const emailSignup = document.body.querySelector( '.' + BASE_CLASS );
 
 if ( emailSignup ) {
-  var language = document.body.querySelector( '.content' ).lang;
-  var formSubmit = new FormSubmit( emailSignup, BASE_CLASS,
-    { validator: fields => {
-        return validators.email( fields.email, '', { language: language } )
-               .msg;
-      },
-      language: language
+  const language = document.body.querySelector( '.content' ).lang;
+  const formSubmit = new FormSubmit( emailSignup, BASE_CLASS,
```

Comment:
If you end up back in this file, could you clean up this set of params by putting each on it's own line like the eslint4 changes the other day? It's really hard to follow.

Reply:
Stuck it in https://github.com/cfpb/cfgov-refresh/pull/3679/commits/8968c7878347bacca9a20a9af60a0487f176f216